diff --git a/pyproject.toml b/pyproject.toml
index 64ada51..3515a9b 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -78,7 +78,9 @@ dependencies = [
     "numexpr >= 2.6.2",
     "packaging",
     "py-cpuinfo",
-    "blosc2 >= 2.3.0",
+    # Either python-blosc2 OR c-blosc2
+    # See: https://github.com/PyTables/PyTables/blob/master/doc/source/usersguide/installation.rst
+    #"blosc2 >= 2.3.0",
     "typing-extensions >= 4.4.0",
 ]
 
